include/attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionDefines.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __CRYPTID_CRYPTID_DEFINES_H
+#define __CRYPTID_CRYPTID_DEFINES_H
+
+#define ABE_CTILDE_SET_NOT_LAST 0
```

Comment:
In this file as I suppose, this defines are related strongly, so it would be better to use an enum instead of multiple defines.

Reply:
Also, when turning these defines into an enum, make sure to be explicit regarding the numbering. Please do not rely on the automatic numeric values assigned to enum constants by the compiler.